File: Codechef/00 Practise/gen.py
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n = 2e5, m = n;
    string s, t;
    for(int i = 0; i < n; i++)
        s.push_back('1');
    for(int i = 0; i < n; i++)
        t.push_back('6');
    cout << n << ' ' << m << '\n';
    cout << s << '\n' << t << '\n';
    for(int i = 0; i < 10; i++)
        cout << 1000 << ' ';
    cout << '\n';
    for(int i = 0; i < 10; i++)
        cout << 1000 << ' ';
    cout << '\n';
}

File: Codechef/46 JAN20A/gen.cpp
#include <bits/stdc++.h>
using namespace std;
#define int long long

signed main(){
	srand(time(0));
	int n = 100;
	std::vector<int> a(n);
	for(int i = 0; i < n; i++)
		a[i] = i + 1;
	random_shuffle(a.begin(), a.end());
	int q = 10;
	cout << n << ' ' << q << '\n';
	for(int i = 0; i < n; i++)
		cout <<a[i] << ' ';
	cout << '\n';
	while(q--){
		cout << rand()%n + 1 << ' ' << rand()%n + 1 << '\n';
	}
}